semgrep_runner_v1.proto

```diff
@@ -0,0 +1,50 @@
+/*
+ * Spec of the Semgrep Runner service utilized in semgrep-proprietary server command.
+ * 
+ * This file has the _v1 suffix to explicitely represent the
+ * version of this protobuf format used by the Semgrep Runner service. 
+ * If you need to extend this file, please check that all changes are backwards compatible.
+ * If a breaking change is introduced (and ensure that it does need to be introduced), then
+ * bump the version number.
+ */
+syntax = "proto3";
+
+package semgrep_runner_v1;
+
+option go_package = "semgrep_runner.v1";
+
+import "protos/semgrep_output_v1.proto";
+
+enum RuleFormat {
+  YAML = 0;
```

Comment:
Nit: let’s follow protobuf best practices with an UNKNOWN = 0 value

Reply:
(maybe in a future PR we could wire up `buf` for proto linting)

Reply:
Good call! Added the `UNSPECIFIED` value and prefix so we don't have to change this later